test/tasks/SingleMultisigTask.t.sol

```diff
@@ -251,41 +248,6 @@ contract SingleMultisigTaskTest is Test {
         localMultisigTask.simulateRun(taskConfigFilePath);
     }
 
-    function testMockDisputeGameWithCodeExceptionsWorks() public {
```

Comment:
I removed these tests as they weren't SingleMultisigs. I added them to the Nested test file and included assertions.